### PR TITLE
Force the media to load for iOS and Safari on desktop

### PR DESCRIPTION
**Pull request by Dananji**
Related issue: #583

### CODE REVIEW

File: src/components/MediaPlayer/VideoJS/VideoJSPlayer.js [hunk 1]
```diff
@@ -527,6 +527,20 @@ function VideoJSPlayer({
...
+       * When either player/browser tab is muted Safari and Chrome in iOS doesn't seem to 
+       * load enough data related to audio-only media for the Video.js instance to play 
+       * on page load.
+       * Since, it is not possible to detect muted tabs in JS the condition avoids
+       * checking for muted state altogether.
+       * Without this, Safari will not reach player.readyState() = 4, the state
+       * which indicates the player that enough data is available on the media
+       * for playback.
+       */
+      if (!isVideo && (IS_SAFARI || IS_IOS) && player.readyState() != 4) {
```
Comment (cjcolvar) [inline src/components/MediaPlayer/VideoJS/VideoJSPlayer.js:541]: Are there more ready states than 4?  If so should this be `player.readyState() < 4`?

**Review by cjcolvar — COMMENTED** [review]
Looks good, but I had one question before approving.

File: src/components/MediaPlayer/VideoJS/VideoJSPlayer.js (hunk 1 above)
Comment (cjcolvar) [reply src/components/MediaPlayer/VideoJS/VideoJSPlayer.js:541]: It looks like ready states only go to 4.
https://docs.videojs.com/player#readyState

**Review by cjcolvar — APPROVED** [review]
:+1:

File: src/components/MediaPlayer/VideoJS/VideoJSPlayer.js (hunk 1 above)
Comment (Dananji) [reply src/components/MediaPlayer/VideoJS/VideoJSPlayer.js:541]: Yes, the media would not play unless it reaches 4.

**Pull request merged**